new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -454,7 +494,7 @@ public static V1PodList listPods(String namespace, String labelSelectors) throws
           );
     } catch (ApiException apex) {
       logger.severe(apex.getResponseBody());
-      throw apex;
+      throw null;
```

Comment:
why are you throwing a null here instead of the original ApiException ?

Reply:
what does it even mean to throw a null :)

Reply:
does it mean you didn't throw at all?

Reply:
if two nulls are thrown in an empty wood, and noone was there to see it.....

Reply:
revert the logic to throw ApiException 

} catch (ApiException apex) {
      logger.severe(apex.getResponseBody());
      throw apex;
}